_data/3raxton/3raxton.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Braxton Huff
+institution: Santa Monica College
+profile_pic: braxton.png 
+quote: Give back and support those who helped you make it to where you are today – braxtonhuff.com
```

Comment:
Change the em dash – to a single dash - as it is a non-ascii character.

Reply:
Thanks for making me aware of this. I've updated it in [this commit](https://github.com/education/graduation/pull/4298/commits/6f9f74cfd0dc339a568d49bf7cea28290b93c782). Please let me know if there is anything else needed!